server/channels/app/export.go

```diff
@@ -869,7 +873,8 @@ func (a *App) exportCustomEmoji(c request.CTX, job *model.Job, writer io.Writer,
 		pathToDir := filepath.Join(outPath, exportDir)
 		if exportFiles {
 			if _, err := os.Stat(pathToDir); os.IsNotExist(err) {
-				os.Mkdir(pathToDir, os.ModePerm)
+				if err := os.Mkdir(pathToDir, os.ModePerm); err != nil {
+					c.Logger().Error("Error creating directory", mlog.String("path", pathToDir), mlog.Err(err))
```

Comment:
I think we should return the error here. If we can't create the export dir, the rest of the method will just fail. Let's be load and clear.